less/mosaico/article-list.less

```diff
@@ -231,3 +233,9 @@
     padding: 30px;
   }
 }
+
+.list-article--hide-mobile {
```

Comment:
A simpler way to do this would be something like (within a media query)
```
.mosaico-asidelist.mosaico-asidelist__latest > ul > li:nth-child(5) ~ * {
  display: none;
}
```